[Bug]: NPM Install Fails

### Issue description

When installing any version newer than rc.3 installation fails

### Version

rc.4 and up

### Expected behaviour

Installation should succeed.

### Actual behaviour

Fails

### Can the issue reliably be reproduced?

Yes

### Steps to reproduce the issue

1. Have Macbook Air M1. Maybe issue is also on other architectures.
2. Install `npm i @iota/client@latest`

Error occures also with 

 v14.20.0
 v16.17.0
 v18.12.1
 
 With the rc.3 I get:
 
 ```sh
 Error: Cannot find module '../../build/Release/index.node'
Require stack:
- /Users/.../dist/main.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
    at Function.Module._load (node:internal/modules/cjs/loader:841:27)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/Users/.../api/node_modules/@iota/client/lib/bindings.ts:7:1)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Function.Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
```

### Errors
[2023-03-07T08_42_00_561Z-debug-0.log](https://github.com/iotaledger/iota.rs/files/10907628/2023-03-07T08_42_00_561Z-debug-0.log)


### Duplicate declaration

- [X] I have searched the issues tracker this issue and there is none

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/.changes/install.md
+++ b/.changes/install.md
@@ -1,0 +1,6 @@
+
+---
+"nodejs-binding": patch
+---
+
+Fix install command;

--- a/client/bindings/nodejs/package.json
+++ b/client/bindings/nodejs/package.json
@@ -21,7 +21,7 @@
     "lint": "eslint --ignore-path .eslintignore --ext .js,.ts .",
     "format": "prettier --ignore-path .eslintignore -w {,*/**/}*.{ts,js,json}",
     "format-check": "prettier --ignore-path .eslintignore -c {,*/**/}*.{ts,js,json}",
-    "install": "prebuild-install --runtime napi --tag-prefix nodejs-binding-v || npm run rebuild",
+    "install": "prebuild-install --runtime napi --tag-prefix nodejs-binding-v && tsc || npm run rebuild",
     "build": "npm run build:neon && tsc",
     "build:neon": "cargo-cp-artifact -ac iota-client-nodejs ./index.node -- cargo build --release --message-format=json-render-diagnostics",
     "rebuild": "npm run build:neon && tsc && node scripts/neon-build.js --strip && node scripts/strip.js",

